tests/subsys/event_manager_proxy/modules/common_utils.c

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <zephyr.h>
+#include <app_event_manager.h>
+
+#include "test_config.h"
+#include "common_utils.h"
+#include "simple_events.h"
+#include "data_events.h"
+#include "test_events.h"
+
+#define MODULE common_utils
```

Comment:
Shouldn't it be in utils folder as it does not use event manager?

Reply:
Well... in some twisted way but it uses Event Manager in the end.